build.rs

```diff
@@ -10,44 +9,9 @@ const PROTO_DIR: &str = "proto";
 const CARGO_OUT_DIR: &str = "proto";
 
 fn main() -> anyhow::Result<()> {
-    ensure_actor_bundle_includable()?;
     generate_protobuf_code()
 }
 
-fn ensure_actor_bundle_includable() -> anyhow::Result<()> {
```

Comment:
Why don't we need this check anymore?

Reply:
It's covered by the `asset_integrity` test now